src/utils.cr

```diff
@@ -0,0 +1,103 @@
+require "./dppm_rest_api"
+
+# A helper method for the select_users method -- splits a comma-separated
+# list of numbers in a string into a `Set` of `Int32` values.
+def split_numbers(number_list : String) : Set(Int32)
```

Comment:
Should be namespaced in CLI, there are only used there.

Reply:
That's not the case anymore. It is used by the `/{user, users,groups}` routes now. That is why these functions have been moved to this file.

To be honest I expected you to be dissatisfied with this solution and meant to ask you for suggestions of how to resolve this issue. Specifically, the `/user` routes need `selected_users`

Reply:
The CLI depends of the whole application, but not actions. They can be put in actions where they are used, or a new module.
Having them globally won't help the understanding and the API docs.